src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Extensions/ArgumentSyntaxExtensions.vb

```diff
@@ -31,12 +31,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
                 Return Nothing
             End If
 
-            Dim invocableExpression = TryCast(argumentList.Parent, ExpressionSyntax)
-            If invocableExpression Is Nothing Then
-                Return Nothing
-            End If
-
-            Dim symbol = semanticModel.GetSymbolInfo(invocableExpression, cancellationToken).Symbol
+            ' Gets the symbol info for the SyntaxNode 
```

Comment:
You probably don't need this comment here since yes that's just what GetSymbolInfo is doing.